install/operator/tests/e2e/install_with_defaults_test.go

```diff
@@ -17,240 +17,242 @@
-
-    if err := f.Client.Update(ctx, syndesis); err != nil {
-        t.Fatalf("unable to update syndesis cr name [%s], got error [%v]", cr, err)
-    }
-
-    // lets make sure that changes are applied to the custom resource before we continue
-    err := wait.Poll(5, 30, func() (done bool, err error) {
-        syndesis := &v1alpha1.Syndesis{}
-        if err := f.Client.Get(ctx, client.ObjectKey{Namespace: namespace, Name: cr}, syndesis); err != nil {
-            t.Logf("unable to fetch syndesis custom resource [%s]", cr)
-            return false, err
-        }
-
-        if syndesis.Spec.Components.Server.Tag == flagtests[0].value {
-            t.Logf("tag [%s] for imagestream [%s] found in syndesis custom resource", flagtests[0].value, flagtests[0].imageName)
-            return true, nil
-        }
-
-        t.Logf("tag [%s] for imagestream [%s] still not found in syndesis custom resource", flagtests[0].value, flagtests[0].imageName)
-        return false, nil
-    })
-    if err != nil {
-        t.Fatalf("syndesis custom resource didn't update correctly: %v", err)
-    }
-
-    // lets give the operator some time to pickup ithe changes
-    t.Log("waiting 30 seconds for the operator to pick up the changes")
-    time.Sleep(30 * time.Second)
-    err = wait.Poll(5, 60, func() (done bool, err error) {
-        is := &imagev1.ImageStream{}
-
-        for _, tt := range flagtests {
-            err = f.Client.Get(goctx.TODO(), client.ObjectKey{Namespace: namespace, Name: tt.imageName}, is)
-            if err != nil {
-                t.Logf("unable to fetch syndesis custom resource [%s]", cr)
-                return false, err
-            }
-
-            t.Logf("imagestream %s has %d tags", tt.imageName, len(is.Spec.Tags))
-            if is.Spec.Tags[0].Name == tt.value {
-                t.Logf("tag [%s] for imagestream [%s] matches [%s]", is.Spec.Tags[0].Name, tt.imageName, tt.value)
-            } else {
-                t.Logf("tag [%s] for imagestream [%s] doesnt match [%s]", is.Spec.Tags[0].Name, tt.imageName, tt.value)
-                return false, nil
-            }
-        }
-
-        t.Log("all imagestreams checked and all conditions were met")
-        return true, nil
-    })
-    if err != nil {
-        t.Fatalf("imagestreams didnt update correctly: %v", err)
-    }
+	t.Log("Changing ImageStreams for syndesis components")
+
+	ctx := goctx.TODO()
+	syndesis := &v1alpha1.Syndesis{}
+	if err := f.Client.Get(ctx, client.ObjectKey{Namespace: namespace, Name: cr}, syndesis); err != nil {
+		t.Fatalf("unable to fetch syndesis cr name [%s], got error [%v]", cr, err)
+	}
+
+	var flagtests = []struct {
+		name      string
+		imageName string
+		value     string
+	}{
+		{"Server", "syndesis-server", "1.8"},
+		{"Meta", "syndesis-meta", "1.8"},
+		{"S2I", "syndesis-s2i", "1.8"},
+		{"UI", "syndesis-ui", "1.8"},
+	}
+
+	syndesis.Spec.Components.Server.Image = flagtests[0].imageName + ":" + flagtests[0].value
```

Comment:
It doesnt make much sense to test the tag change, but rather the whole image reference

Reply:
I've not been able to run the e2e tests..  :(

Reply:
this is the command I use

```
GO111MODULE=on operator-sdk test local ./tests/e2e --debug --up-local --namespace syndesis-test --local-operator-flags "run --operator-config=build/conf/config.yaml" --go-test-flags "-v"
```

change the namespace